packages/contentful/src/crn/queries/analytics.queries.ts

```diff
@@ -146,11 +145,10 @@ export const FETCH_TEAM_PRODUCTIVITY = gql`
         displayName
         inactiveSince
         linkedFrom {
-          researchOutputsCollection(limit: 100) {
+          researchOutputsCollection(limit: 600) {
```

Comment:
These are the teams that have most research outputs in prod
<img width="345" alt="Screenshot 2024-04-23 at 08 24 55" src="https://github.com/yldio/asap-hub/assets/16595804/2ffb3d88-775d-4c10-8402-f7fe4f7086de">

The max currently is 470, but I've made the limit to be 600 randomly.
![Screenshot 2024-04-23 at 08 25 02](https://github.com/yldio/asap-hub/assets/16595804/db9e3d6f-1581-418f-9293-d45a53f8f487)